packages/app-admin-ui/client/pages/Home/components.js

```diff
@@ -76,7 +75,7 @@ export const Name = styled.span(
   font-size: 1.1em;
   font-weight: 500;
   max-width: 100%;
-  overflow: hidden;
+  overflow-x: hidden;
   text-overflow: ellipsis;
   transition: border-color 80ms linear;
   white-space: nowrap;
```

Comment:
```suggestion
  white-space: nowrap;
  padding-right: 2em;
```

Reply:
Allows text ellipsis to work

Reply:
Added. Needed to switch to a div for it to work, though (and I still needed `overflow-x`).

Reply:
Add `box-sizing: border-box` rather than changing it to a div.

Reply:
Done.